docs/arf.md

```diff
@@ -1711,13 +1711,23 @@ purposes:
     Wallet Instance. The WTE is described in \[Topic 9\]. The WTE has
     two main purposes:
 
+    -   It describes the capabilities and properties of the Wallet
+        Instance, the User device and the WSCD(s) and lets the Wallet Provider rerecognize its Wallet Instance.
+
+    -   Moreover, the WTE contains a WTE public key. This key can be used
+        to authenticate the Wallet Instance towards the Wallet Provider.
+
+3.  The EUDI Wallet Provider issues a Issuer Trust Evidence (ITE) to the
```

Comment:
shouldn't this be an ITE per issuer (at least)? Depending on the strategy it could also be an ITE per PID/attestation to be issued.